docker/run.sh

```diff
@@ -1,2 +1,2 @@
 echo "Starting container"
-docker start -a sd
\ No newline at end of file
+docker start --gpus all sd
```

Comment:
--gpus flag is not a valid `docker start` argument, but a `docker run`
If it does work for you though, please add a failover command, e.g. `docker start --gpus all sd || docker start -a sd `

Reply:
So, this is leftovers from testing, there is no ```--gpus all```. The reason to add it due to this https://github.com/rgryta/Stable-Diffusion-WSL2-Docker/issues/11